examples/dft_plus_u/STRU

```diff
@@ -0,0 +1,35 @@
+ATOMIC_SPECIES
+Ni1 1.000 ./Ni_ONCV_PBE-1.0.upf
```

Comment:
Under the test cases rules, pseudopotential and orbital files should move to path "abacus-develop/tests/PP_ORB/"